### ISSUE TITLE
Logo design

### ISSUE THREAD

**Issue by Secret-chest** [StractOrg/stract#187]
Note that I'm not a designer.

Currently I see the logo is just text. Is it a placeholder?

Also, what does the name mean?

**Comment by mikkeldenker** [comment]
The name is actually quite an interesting story. I knew I wanted to have a short name where the .com domain was available and one that could be pronounced. It turns out it’s very difficult to come up with one of these, so I actually ended up writing a script that would first generate a list of random words with at most $k$ letters. Then I trained a naive bayes classifier on bigrams and trigrams over a big list of english words and some randomly scrambled words to detect if a string could be pronounced. I then filtered the randomly generated list using this classifier and looked up if the .com domain was available.

This left me with a pretty big list of names that I then went through manually until Stract stood out. Of course it helps that it sounds a bit like ‘extract’ as in ‘extract information from the web’ or something like that.

The logo is just the name written in the montserrat font, but it is not a placeholder.

I'll close this issue as there is no action for us to take.
